doc/api/resources/sendmail_rules.rst

```diff
@@ -27,6 +27,10 @@ restrict_to_status                    list                       List of order s
                                                                  ``n__valid_if_pending`` for payment pending but already confirmed,
                                                                  and ``n__pending_overdue`` for pending with payment overdue.
                                                                  The default is ``["p", "n__valid_if_pending"]``.
+checked_in_status                     string                     Checked in states that can be used to filter recipients. Valid strings
+                                                                 are: ``all`` (no filtering, default), ``checked_in`` for
+                                                                 sending mails to checked in customers, and ``no_checkin`` to send mails
+                                                                 to all customers who have not checked in.
```

Comment:
Not really an issue, I would make the wording and formatting (use of paretheses) more consistent.

```suggestion
checked_in_status                     string                     Checked in status to restrict recipients to. Valid strings
                                                                 are: ``all`` for no filtering (default), ``checked_in`` for limiting to
                                                                 attendees that are or have been checked-in, and ``no_checkin`` for
                                                                 limiting to attendees who have not checked-in.
```